### PR TITLE
Update 2017 Beam-pipe Middle Section

### PR DESCRIPTION
**Pull request by ianna**
* Middle section material changed from AlBe to Al 2219

The material of the outer part of the current central beam pipe is Al 2219. AlBe which was used in early studies of the beam pipe was finally discarded for cost and production risk reasons.

* Restructure of beam-pipe files as suggested by @davidlange6 
* Update scenarios accordingly

### CODE REVIEW

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2018XML_cfi.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
         'Geometry/CMSCommonData/data/muonBase.xml',
         'Geometry/CMSCommonData/data/cmsMuon.xml',
         'Geometry/CMSCommonData/data/mgnt.xml',
-        'Geometry/CMSCommonData/data/PhaseI/beampipe.xml',
+        'Geometry/CMSCommonData/data/beampipe/2015/v1/beampipe.xml',
```
Comment (kpedro88) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2018XML_cfi.py]: Should this be 2017/v1?

Comment (ianna) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2018XML_cfi.py]: yep, thanks

**Comment by kpedro88** [comment]
+1

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by smuzaffar** [comment]
@Ianna, I guess this broken relval in IB
```
----- Begin Fatal Exception 14-Mar-2017 05:24:55 CET-----------------------
An exception of category 'FileInPathError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESSource: class=XMLIdealGeometryESSource label='XMLIdealGeometryESSource_CTPPS'
Exception Message:
edm::FileInPath unable to find file Geometry/CMSCommonData/data/PhaseI/beampipe.xml anywhere in the search path.
The search path is defined by: CMSSW_SEARCH_PATH
${CMSSW_SEARCH_PATH} is: /build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_1_X_2017-03-13-2300/src:/build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_1_X_2017-03-13-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/nweek-02463/slc6_amd64_gcc530/cms/cmssw/CMSSW_9_1_X_2017-03-13-2300/src:/cvmfs/cms-ib.cern.ch/nweek-02463/slc6_amd64_gcc530/cms/cmssw/CMSSW_9_1_X_2017-03-13-2300/external/slc6_amd64_gcc530/data
Current directory is: /build/cmsbld/jenkins-workarea/workspace/ib-run-relvals/CMSSW_9_1_X_2017-03-13-2300/pyRelval/5.1/5.1_TTbar+TTbarFS+HARVESTFS
----- End Fatal Exception -------------------------------------------------
```

**Comment by ianna** [comment]
@smuzaffar - thanks, I'm looking into it.

**Comment by ianna** [comment]
@jan-kaspar - it looks like XMLIdealGeometryESSource_CTPPS adds some files which are not needed for its description. Especially, a beam pipe should not be needed at RECO step. Would you like me to trim the list down or would you prefer to do it yourself? Thanks.

**Comment by jan-kaspar** [comment]
@ianna Please do it. Thanks!

**Comment by ianna** [comment]
This is a temporary fix:
https://github.com/cms-sw/cmssw/pull/17914

**Pull request merged**